Scripts/Python/ki/__init__.py

```diff
@@ -2139,10 +2139,16 @@ def WearKILevel(self, level):
             if gender > kFemaleClothingGroup:
                 gender = kMaleClothingGroup
             avatar.netForce(1)
-            if gender == kFemaleClothingGroup:
-                avatar.avatar.wearClothingItem("FAccPlayerBook")
+            if StartInCleft():
+                if gender == kFemaleClothingGroup:
+                    avatar.avatar.removeClothingItem("FAccPlayerBook")
+                else:
+                    avatar.avatar.removeClothingItem("MAccPlayerBook")
             else:
-                avatar.avatar.wearClothingItem("MAccPlayerBook")
+                if gender == kFemaleClothingGroup:
+                    avatar.avatar.wearClothingItem("FAccPlayerBook")
+                else:
+                    avatar.avatar.wearClothingItem("MAccPlayerBook")
```

Comment:
I think this should be written more tersely, eg:
```python
            playerBook = "FAccPlayerBook" if gender == kFemaleClothingGroup else "MAccPlayerBook"
            if StartInCleft():
                avatar.avatar.removeClothingItem(playerBook)
            else:
                avatar.avatar.wearClothingItem(playerBook)
```

Unfortunately, GitHub won't let me make a suggestion across a deletion, so here it is. BTW old Pythons don't allow single line if statements, so you'll have to break that into multiple lines if you're porting this to any obsolete codebases.